<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoProcManager</string>
  </property>
  <widget class="QPushButton" name="pBtnStopTasks">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Tasks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRunTasks">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Tasks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRebootTasks">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reboot Tasks</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkInfiniteStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Infinite Restart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRebootZmqBroker">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reboot ZmqBroker</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRebootTCPServer">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reboot TCPServer</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>421</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Process</string>
    </attribute>
    <widget class="QPlainTextEdit" name="edProcessStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>zmq_tcp_server</string>
    </attribute>
    <widget class="QPlainTextEdit" name="edZmqTCPServerStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>zmq_broker</string>
    </attribute>
    <widget class="QPlainTextEdit" name="edZmqBrokerStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
